crates/cache/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "spin-cache"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+anyhow = "1.0"
+dirs = "4.0"
+dunce = "1.0"
+futures = "0.3.14"
+itertools = "0.10.3"
+lazy_static = "1.4.0"
+mime_guess = { version = "2.0" }
+oci-distribution = { git = "https://github.com/radu-matei/oci-distribution", branch = "update-dockerhub-default-registry" }
+regex = "1.5.4"
+reqwest = "0.11"
+semver = "1.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tempfile = "3.3.0"
+thiserror = "1.0.37"
+tokio = { version = "1.20", features = ["full"] }
+tracing = "0.1"
```

Comment:
`tracing` is also a workspace dependency; should/could we use that here? `tracing = { workspace  = true }`